services/app-api/src/postgres/contractAndRates/prismaContractRateAdaptors.ts

```diff
@@ -0,0 +1,294 @@
+import type {
+    ContractFormEditableType,
+    RateFormEditableType,
+} from '../../domain-models/contractAndRates'
+import type { GenericDocument } from '../../gen/gqlServer'
+import { emptify, nullify } from '../prismaDomainAdaptors'
+
+// ADD RETURN TYPES FROM PRISMA SEE "includes" "satifies"
+// contractTableFullPayload - contractTableCreateArgs / contract
```

Comment:
FYI @macrael I did try adding return types here like  `Prisma.ContractRevisionTableCreateInput` or a partial of that type but it was resulting in lots of lint errors. Even when I resolved those (about 40 hours) I was seeing test failures. Did not feel confident at all in the change. Also looked at using `satisfies` keyword and Prisma.validator and got a bit confused. Maybe you could take a look? 

Either way can I get a 👍 that this is good to merge without the types in place for this spirnt